velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -1045,7 +1045,6 @@ bool SubstraitToVeloxPlanValidator::validate(
   std::unordered_set<std::string> supportedFuncs = {
       "sum",
       "sum_merge",
-      "bin",
```

Comment:
Thank you! Could you help add some comments to this variable to avoid future misleading? For example: The supported aggregation functions.

Reply:
Thanks! already updated.